src/main/kotlin/org/jetbrains/research/ml/ast/gumtree/tree/PsiTreeConverter.kt

```diff
@@ -18,27 +20,34 @@ object PsiTreeConverter {
      *
      *  On each iteration children from PSI are converted to GumTree format and added to GumTree parents one by one;
      *  their PSI children are added to the corresponding place in the PSI children collection.
+     *
+     *  Ignore nodes with WHITE_SPACE_TYPE
```

Comment:
Could you please add an explanation of why we need to ignore such nodes? We will surely forget about it after some time

Reply:
Also, shouldn't it be an option, so the user can decide whether they want to ignore them? It seems that we can keep whitespaces if we don't use serialization of gumtree trees (as I understand), so it's a little bit strange to ignore them with no choice provided